docs/connect-blockchain/multi-chain.mdx

```diff
@@ -114,6 +114,8 @@ the Ethereum provider.
     { label: "Binance Smart Chain", value: "bsc" },
     {label: "Avalanche", value: "avalanche"},
     {label: "Cronos", value: "cronos"},
+    {label: "Flare", value: "flare"},
```

Comment:
Unfortunately we cannot add this in the multi-chain docs. That is particular to an example that we have and we do not want to change it.